src/incubator/WheelPicker/index.tsx

```diff
@@ -235,14 +265,19 @@ const WheelPicker = React.memo(({
             onLayout={scrollToPassedIndex}
             // @ts-ignore
             ref={scrollView}
+            // @ts-expect-error
             contentContainerStyle={contentContainerStyle}
             snapToInterval={itemHeight}
             decelerationRate={Constants.isAndroid ? 0.98 : 'normal'}
             renderItem={renderItem}
             getItemLayout={getItemLayout}
             initialScrollIndex={currentIndex}
+            onContentSizeChange={contentWidth => setFlatListWidth(contentWidth)}
           />
         </View>
+      </View>
+      {/* @ts-expect-error */}
+      <View style={labelContainerStyle} width={flatListWidth} pointerEvents="none">
```

Comment:
Maybe worth including the wrapping View of the label container in the label container code